dirs/dirs.go

```diff
@@ -110,7 +110,6 @@ var (
 	SystemLocalFontsDir       string
 	SystemFontconfigCacheDirs []string
 
-	FreezerCgroupDir string
 	PidsCgroupDir    string
```

Comment:
will this one move too at some point?

Reply:
@pedronis yes, it goes away entirely in the next few branches.